Resources/Private/CodeTemplates/Extbase/Configuration/TCA/tableName.phpt

```diff
@@ -9,7 +9,6 @@ return [
         'sortby' => 'sorting',</f:if><f:if condition="{extension.supportVersioning}">
         'versioningWS' => true,</f:if><f:if condition="{extension.supportLocalization}">
         'languageField' => 'sys_language_uid',
-        'transOrigPointerField' => 'l10n_parent',
```

Comment:
Why removing this field?

Reply:
I misread the deprecation warning - 
" which is no longer needed and should therefore be removed"
but that relates only to defining it as exclude field...